src/introduction/introduction.tex

```diff
@@ -0,0 +1,16 @@
+% textidote: ignore begin
+\chapter{Introduction}\label{ch:introduction}
+% textidote: ignore end
+
+Learning is something we do every day.
+It is something most people enjoy and strive to experience.
+However, learning can also be difficult.
+In this report we will explore the concept of collaborated assisted learning and how software can assist in this method.
+We will be focusing our analysis around the age-old game of chess.
+
+AI and chess engine popularity has risen greatly in the past years and have been developed to a point that they
+outperform humans.
```

Comment:
Source here would be nice

Reply:
It is just the introduction, why it does not require sources I don't think. Furthermore the point will be sourced in the problem analysis later, why I don't think it is necessary to include sources in the introduction